<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MCComputer" name="com.vnt.DemoMavenProject01.myTest.LoginTest" tests="2" failures="0" timestamp="19 Nov 2020 18:00:12 GMT" time="32.841" errors="1">
  <testcase name="verifyInfoRegistrationDisplayed" time="12.427" classname="com.vnt.DemoMavenProject01.myTest.LoginTest"/>
  <testcase name="doLogin" time="20.414" classname="com.vnt.DemoMavenProject01.myTest.LoginTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.vnt.DemoMavenProject01.myPages.RegisterPage.gotoBankProject(RegisterPage.java:87)
at com.vnt.DemoMavenProject01.myTest.LoginTest.doLogin(LoginTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- doLogin -->
</testsuite> <!-- com.vnt.DemoMavenProject01.myTest.LoginTest -->
